### ISSUE TITLE
Invalid code snipped for prop

### ISSUE THREAD

**Issue by dsoronda** [atom/language-csharp#17]
```
  'Property':
    'prefix': 'prop'
    'body': '${1:string} ${2:PropertyName} { get; set; }'
```

Property is public so body should be like :
```
    'body': 'public ${1:string} ${2:PropertyName} { get; set; }'
```

**Comment by shiftkey** [comment]
Properties do not necessarily need to be public to be valid C#:

```
object MyCoolProperty { get; set; }
protected object MyCoolProperty { get; set; }
```

The access modifiers can also exist alongside the `get` and `set`:

```
public object MyCoolProperty { get; private set; }
public object MyOtherProperty { get; protected set; }
```

**Comment by dsoronda** [comment]
True, but, Visual Studio/Xamarin Studio/ShardDevelop by default use prop snippet to generate public property.
So following common sense from others IDE, same should be applied here.

**Comment by shiftkey** [comment]
@dsoronda my apologies - I thought we were talking about syntax highlighting.

If you'd like to submit a PR to make that change, I think it'd be approved...

**Comment by dsoronda** [comment]
i created PR 22 days ago and still nothing. I even created another PR for bad "w" and "wl" snippet.

**Comment by shiftkey** [comment]
> i created PR 22 days ago and still nothing.

Was it this PR? https://github.com/dsoronda/language-csharp/pull/1 Unfortunately it wasn't targeting the fork instead of this repository, so we didn't see it! Sorry!

**Comment by 50Wliu** [comment]
Unless I'm missing something, I think that this was fixed by #18, so closing.
